### PR TITLE
Add link to educator info/learning outcomes in skillmap

### PR DESCRIPTION
**Pull request by shakao**
only allow links to our docs or github, at the moment

image
image

@Jaqster @kiki-lee

### CODE REVIEW

File: skillmap/src/lib/skillMapParser.ts [hunk 1]
```diff
@@ -360,6 +360,19 @@ function getContrastingColor(color: string) {
     }
 }
 
+function cleanInfoUrl(url?: string) {
+    // No info URL provided
+    if (!url) return undefined;
+
+    // Valid URL to Github (eg a README)
+    if (url.match(/^(https?:\/\/)?(www\.)?github\.com/gi)) return url.replace(/\?[\s\S]+$/gi, "");
```
Comment (riknoll) [inline skillmap/src/lib/skillMapParser.ts:368]: this could match `https://www.github.completelyUnsafeWebsite.com`

**Review by riknoll — CHANGES_REQUESTED** [review]

**Review by riknoll — APPROVED** [review]

**Comment by kiki-lee** [comment]
I'd prefer it to be at the bottom…beneath all of the current info…but I wouldn't stop the PR for that.

**Comment by shakao** [comment]
ah!! i just realized i implemented the wrong one, sonia also prefered it below the map description), let me make that change really quick

**Comment by shakao** [comment]
@kiki-lee 
image
image

**Review by abchatra — APPROVED** [review]

**Comment by Jaqster** [comment]
Looks good to me!

**Pull request merged**